src/vs/workbench/browser/parts/globalCompositeBar.ts

```diff
@@ -523,6 +555,16 @@ export class AccountsActivityActionViewItem extends AbstractGlobalActivityAction
 		}
 	}
 
+	private updateActionItem(): void {
```

Comment:
Should not this method has to be called when
- created
- profile changed
- avatar state changed